code/modules/mob/living/carbon/xenomorph/strains/castes/runner/acid.dm

```diff
@@ -83,7 +83,10 @@
 		modify_acid(acid_slash_regen_standing)
 
 /datum/behavior_delegate/runner_acider/on_life()
-	modify_acid(acid_passive_regen)
+	if(acid_amount > 400)
+		return
```

Comment:
You haven't tested this; this will break all the code below it because of this early return